api/v1beta1/flowcollector_types.go

```diff
@@ -146,11 +146,17 @@ type FlowCollectorIPFIX struct {
 	OVNKubernetes OVNKubernetesConfig `json:"ovnKubernetes,omitempty" mapstructure:"-"`
 }
 
-type FeatureConfigType string
+// Agent feature, can be one of:<br>
+// - `PKT_DROP`, to track packet drops.<br>
+// - `DNS_TRACKING`, to track specific information on DNS traffic.<br>
+// - `FLOW_RTT`, to track L4 latency. <i>Unsupported (*)</i><br>
```

Comment:
pls remove <br> and other weired chars at the end of the above lines

Reply:
@msherif1234 I guess you are talking about the HTML blocks right ? That's not new to this PR: https://issues.redhat.com/browse/NETOBSERV-1104 ; it's related to asciidocs
Let's keep this as is and solve it in the related ticket